contrib/ci/oss-fuzz.py

```diff
@@ -26,7 +26,15 @@ def __init__(self) -> None:
             "OUT", os.path.realpath(os.path.join(DEFAULT_BUILDDIR, "out"))
         )
         self.srcdir = self._ensure_environ("SRC", os.path.realpath(".."))
-        self.ldflags = ["-lpthread", "-lresolv", "-ldl", "-lffi", "-lz", "-llzma"]
+        self.ldflags = [
+            "-lpthread",
+            "-lresolv",
+            "-ldl",
+            "-lffi",
+            "-lz",
+            "-llzma",
+            "-lgnutls",
```

Comment:
ohh? I thought we wanted to keep this out the fuzzer static binaries to decrease the size?

Reply:
Oh, I didn't see a comment for that anywhere and so I didn't remember this.  I'll fix it.